derived_object_msgs/Object object
float32 distance
float32[] swipe
float32 likelihood
bool is_touched
bool is_interaction
bool is_front
bool is_important
